backend/src/cache/deployer/Dockerfile

```diff
@@ -1,11 +1,17 @@
-FROM gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine
+FROM alpine
```

Comment:
@hbelmiro gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine image is not supporting multiple architecture. It only amd/arm architectures.

 Since gcloud component installed is only kubectl (https://github.com/kubeflow/pipelines/blob/master/backend/src/cache/deployer/Dockerfile#L8) we are taking plain vanila alpine and installing all the needed packages including kubectl from binary distributions as you see in below comment.

Reply:
@dilipgb I see. But I'm not sure how able we are to change images without a broader involvement of the community. Like asking in the community call, for example.

cc @HumairAK